src/components/docs/MultiStepSnippet/styles.module.css

```diff
@@ -1,6 +1,9 @@
 @value small-tablet-breakpoint-query from '~libra-docusaurus/src/variables.module.css';
+@value small-tablet-breakpoint-max-query from '~libra-docusaurus/src/variables.module.css';
```

Comment:
Basically when the headers are above the snippet (mainly on smaller widths), this makes sure the headers are at least a square, or longer if there is overflow
<img width="461" alt="Screen Shot 2020-06-14 at 11 59 21 PM" src="https://user-images.githubusercontent.com/20331238/84622726-f2cded80-ae9a-11ea-8a43-e4f150fdd300.png">